pkg/virt-controller/watch/preset_test.go

```diff
@@ -111,12 +111,12 @@ var _ = Describe("VM Initializer", func() {
 		})
 	})
 
-	Context("Conflict detection", func() {
+	Context("Override detection", func() {
 		var vm v1.VirtualMachine
 		var preset v1.VirtualMachinePreset
 		truthy := true
 		falsy := false
-		recorder := &FakeRecorder{}
+		var recorder *FakeRecorder
```

Comment:
recorder.Events should probably be inspected to verify the expected type/number of events occur with conflicts. look at the ovm_test.go file for examples.

Reply:
It would be worth deciding if we're going to keep checking and reporting preset overrides since most of the test cases involving a recorder will be affected.